hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/HealthTransformations.scala

```diff
@@ -64,14 +65,109 @@ object HealthTransformations {
             categorical
           )
       }
-    } else {
-      None
-    }
+    } else None
 
-  // list conditions and assign categorical Longs
-  val infectiousDiseaseCondition = 1L
+  def mapCongenitalEyeDisorder(rawRecord: RawRecord): Iterable[HlesHealthCondition] =
+    if (rawRecord.getBoolean("hs_congenital_yn") && rawRecord.getBoolean("hs_cg_eye_disorders_yn"))
```

Comment:
does the hs_congenital_yn flag show up for non eye congential disorders?

Reply:
Yes